rpm/tarantool-expirationd.spec

```diff
@@ -29,6 +31,10 @@ make test
 %install
 install -d %{buildroot}%{_datarootdir}/tarantool/
 install -m 0644 expirationd.lua %{buildroot}%{_datarootdir}/tarantool/
+curl -L https://tarantool.io/avEfjSi/release/1.10/installer.sh | bash
+yum install -y tarantool
```

Comment:
1. Use `rpm/prebuild.sh` file (and `debian/prebuild.sh` file) to prepare build environment. packpack calls it before rpmbuild / debuild. Using of a network within a package build script is generally discouraged.
2. You don't need to install tarantool explicitly, it is in the build dependencies.

Reply:
new tests will run in gh ci